From 1a3f16ded122bf3928aeb9beb71b594bf24ce01a Mon Sep 17 00:00:00 2001
From: orbea <orbea@riseup.net>
Date: Fri, 10 May 2019 14:47:36 -0700
Subject: [PATCH] Fix --enable-opengles and --enable-opengles3 (#8749)

* qb: Fix OpenGLES build.

The gl core video driver requires at least OpenGLES 3 and
OpenGL 1 is not compatible with OpenGLES 2 or 3.
---
 qb/config.libs.sh |  8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/qb/config.libs.sh b/qb/config.libs.sh
index 17eb7ccf20..510386e44c 100644
--- a/qb/config.libs.sh
+++ b/qb/config.libs.sh
@@ -351,6 +351,14 @@ fi
 if [ "$HAVE_OPENGL" = 'no' ] && [ "$HAVE_OPENGLES3" = 'no' ]; then
    die : 'Notice: OpenGL and OpenGLES3 are disabled. Disabling HAVE_OPENGL_CORE.'
    HAVE_OPENGL_CORE='no'
+elif [ "$HAVE_OPENGLES" != 'no' ] && [ "$HAVE_OPENGLES3" != 'yes' ]; then
+   die : 'Notice: OpenGLES2 is enabled. Disabling the OpenGL core driver.'
+   HAVE_OPENGL_CORE='no'
+fi
+
+if [ "$HAVE_OPENGLES" != 'no' ] || [ "$HAVE_OPENGLES3" != 'no' ]; then
+   die : 'Notice: OpenGLES is enabled. Disabling the OpenGL1 driver.'
+   HAVE_OPENGL1='no'
 fi
 
 if [ "$HAVE_ZLIB" = 'no' ]; then
